### PR TITLE
chore(curriculum): provide further info on unordered lists

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Related to #47554

### CODE REVIEW

**Comment by bbsmooth** [comment]
I could be wrong here, but I thought the complaint about this step was that the solution requires `list-style: none` and the `list-style` property hasn't been introduced yet. I think if we are going to fix the instructions for this step it should include an explanation of `list-style`.

**Comment by a2937** [comment]
Oh. I'll get right on that.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ dashedName: step-61
 
 # --description--
 
+While `ul`/`li` elements are great at providing bullets for list items, our radio buttons will instead be used to mark each list item. We
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:10]:
```suggestion
While `ul`/`li` elements are great at providing bullets for list items, your radio buttons will instead be used to mark each list item. You
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ dashedName: step-61
...
+can control what the bullets look by using the `list-style` css property.  For example you can turn your bullets into the circles
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:11]:
```suggestion
can control what the bullets look by using the `list-style` css property. For example you can turn your bullets into the circles
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 3]
```diff
@@ -7,6 +7,16 @@ dashedName: step-61
...
+with the following. 
+
+```css
+.circle-bullet {
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:15]:
```suggestion
ul {
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by moT01 — COMMENTED** [review]
I think this will work with @naomi-lgbt's suggestions - I tried to come up with some a little more concise:

````
While `ul`/`li` elements are great at providing bullets for list items,
your radio buttons don't need them. You can control what the bullets
look with the `list-style` property. For example you can turn your
bullets into circles with the following:
 
 ```css
.ul {
  list-style: circle;
}
```

Remove the default styling for the `.answers-list` items by settings
its style to `none`, and remove the unordered list padding.
````

It's not much different. I condensed a few words and added the specific value to use (`none`) in the instructions. I think I like it a little better, but it works either way.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md (hunk 3 above)
Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:15]: Applied.

**Comment by a2937** [comment]
Thank you for your suggestions.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 4]
```diff
@@ -5,9 +5,19 @@ challengeType: 0
 dashedName: step-61
 ---
 
-# --description--
+While `ul`/`li` elements are great at providing bullets for list items,
+your radio buttons don't need them. You can control what the bullets
+look with the `list-style` property. For example you can turn your
+bullets into circles with the following:
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:11]:
```suggestion
# --description--

While `ul`/`li` elements are great at providing bullets for list items, your radio buttons don't need them. You can control what the bullets look with the `list-style` property. For example you can turn your bullets into circles with the following:
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 5]
```diff
@@ -5,9 +5,19 @@ challengeType: 0
...
 
-Remove the default styling for the list items of `.answers-list`, and remove the unordered list padding.
+```css
+.ul {
+  list-style: circle;
+}
+```
+
+Remove the default styling for the `.answers-list` items by settings
+its style to `none`, and remove the unordered list padding.
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:20]:
```suggestion
Remove the default styling for the `.answers-list` items by settings its style to `none`, and remove the unordered list padding.
```

**Review by moT01 — CHANGES_REQUESTED** [review]
My suggestions add the `# --description--` header back and put the whole description on one line.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md [hunk 6]
```diff
@@ -7,7 +7,15 @@ dashedName: step-61
 
 # --description--
 
-Remove the default styling for the list items of `.answers-list`, and remove the unordered list padding.
+While `ul`/`li` elements are great at providing bullets for list items, your radio buttons don't need them. You can control what the bullets look with the `list-style` property. For example you can turn your bullets into circles with the following:
+
+```css
+.ul {
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148defa9c01520fb9d178a0.md:13]:
```suggestion
ul {
```

Nitpick - element selector, not class.

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @a2937 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**